core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -1298,11 +1298,42 @@ private static String toSql(RelNode root, SqlDialect dialect) {
 
   @Test public void testFullJoinOnTrueCondition() {
     String query = "select * from \"department\"\n"
-        + "FULL JOIN \"employee\" ON TRUE";
+        + "FULL JOIN \"employee\" ON true";
     String expected = "SELECT *\n"
         + "FROM \"foodmart\".\"department\"\n"
         + "FULL JOIN \"foodmart\".\"employee\" ON TRUE";
-    sql(query).ok(expected);
+    String mssql = "SELECT *\n"
+        + "FROM [foodmart].[department]\n"
+        + "FULL JOIN [foodmart].[employee] ON 1 = 1";
+    String oracle = "SELECT *\n"
+        + "FROM \"foodmart\".\"department\"\n"
+        + "FULL JOIN \"foodmart\".\"employee\" ON 1 = 1";
+    sql(query)
+        .ok(expected)
+        .withMssql()
+        .ok(mssql)
+        .withOracle()
+        .ok(oracle);
+  }
+
+  @Test public void testFullJoinOnFalseCondition() {
+    String query = "select * from \"department\"\n"
```

Comment:
They seems strive to using uppercase for reserved words.

Reply:
actually lots of the other tests that surround this one use lowercase for the query